<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailedInfoForm</class>
 <widget class="QWidget" name="DetailedInfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#DetailedInfoForm{
	 background-color: #84AF9B;
	
	
}
</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{



}

QPushButton:hover{


	border:1px solid blue;
	background-color: #FFFFFF;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bookImage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>211</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string>图像</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bookName">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>311</x>
     <y>51</y>
     <width>441</width>
     <height>66</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>99</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	color : rgb(32,36,46);
	font-size: 50px;
	font-weight:800; 
}</string>
   </property>
   <property name="text">
    <string>名字</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_price">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>381</x>
     <y>154</y>
     <width>291</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	color : rgb(197,31,31);
	font-size: 25px;
	font-weight:800; 
}</string>
   </property>
   <property name="text">
    <string>价格</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_author">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>371</x>
     <y>126</y>
     <width>271</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光细圆_CNKI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>作者</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_type">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>87</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	 
	color : rgb(31,36,46);;
	font-size: 40px;
	font-weight:700; 
}</string>
   </property>
   <property name="text">
    <string>类型</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="TE">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>461</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#TE{



}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>313</x>
     <y>154</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>价格 ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>313</x>
     <y>126</y>
     <width>51</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光细黑_CNKI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>作者：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_shoppingcart">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: gray;
	background-color:#DCDCDC;

}


QPushButton:hover{


color:#FFFFF0;
background-color: rgb(174, 16, 39);

}</string>
   </property>
   <property name="text">
    <string>加入购物车</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>530</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
